app/controllers/users/friends_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Users::FriendsController < ApplicationController
+  def index
+    current_user = params["user_id"].to_i
+    weekly_activity = WeeklyActivity.where("user_1_id = ? OR user_2_id = ?", current_user, current_user).first
```

Comment:
```suggestion
    weekly_activity = WeeklyActivity.where(user_1_id: user_id).or(
      WeeklyActivity.where(user_2_id: user_id)
    ).first
```

I personally prefer to use ActiveRecord's API instead of writing SQL as much as possible, but both are fine.

Reply:
yes I was also trying to do it that way, but it wasn't working, I was probably typing it wrong.
Also the SQL seemed more simple so I ended up using it. But already changed it :muscle: thanks